docs/docs/templates.md

```diff
@@ -16,17 +16,22 @@ Sceptre uses CloudFormation or Troposphere templates to launch AWS Stacks. Conve
     └── vpc.py
 ```
 
-Note that as a path to the template is supplied in a stack's Stack Config file, templates may be stored at any arbitrary location on disk.
+Note that as a path to the template is supplied in a Stack Config file, templates may be stored at any arbitrary location on disk.
 
 
 ## CloudFormation
 
-Templates with `.json` or `.yaml` extensions are treated as CloudFormation templates. They are read in and launched without modification.
+Templates with `.json` or `.yaml` extensions are treated as CloudFormation templates. They are read in and used without modification.
 
 
-## Troposphere
+## Jinja
 
-Templates with a `.py` extension are treated as Troposphere templates. They should implement a function named `sceptre_handler(sceptre_user_data)` which returns the CloudFormation template as a `string`. Sceptre User Data is passed to this function as an argument. If Sceptre User Data is not defined in the Stack Config file, Sceptre passes an empty `dict`.
+Templates with `.j2` extensions are treated as Jinja2 templates. Sceptre User Data is accessible within templates as `sceptre_user_data`. For example `{{ sceptre_user_data.some_variable }}`.
```

Comment:
Can we mention that `.j2` files are normal `json`/`yaml` templates with added support for SUD variable injection?